docs/API/REST-V2-Usage.md

```diff
@@ -1040,6 +1040,7 @@ exception of project variables, task annotation settings, workstation settings,
 |--------------|--------|-------------------------------------------|
 | enabled      | bool   | Enable/disable the commit queue           |
 | merge_method | string | method of merging (squash, merge, rebase) |
+| merge_queue  | string | merge queue to use (EVERGREEN or GITHUB ) |
```

Comment:
I think lowercase follows the conventions better, e.g., "squash", from above.

Reply:
Sadly this is consequence of how enums are defined in all caps in GraphQL, which trickles down into the API 😔 (the `merge_method` field above is defined as `String` rather than `Enum` which is why those fields are lowercase)

I confirmed with another UI engineer that the practice right now is to just keep the uppercase form in the API route. But I do agree it looks weird, so I'm going to spend some time thinking about if there's anything we can change about it in the future!